src/components/PeerSelector/PeerSelector.vue

```diff
@@ -98,5 +98,11 @@ export default class PeerSelector extends PeerSelectorTs {}
   .ivu-poptip-body-content {
     overflow: hidden;
   }
+  .ivu-poptip-title {
```

Comment:
hey,teamate!
i will suggest that if you want to change the iview styles.please add use add `scoped`,like this `<style lang="less" scoped></style>`

Reply:
ACK